README.md

```diff
@@ -35,7 +35,7 @@ A list of available kernels can be seen with::
 
 ## Resources
 - [Project Jupyter website](https://jupyter.org)
-- [Documentation for Jupyter Console](http://jupyter-console.readthedocs.org/en/latest/) [[PDF](https://media.readthedocs.org/pdf/jupyter-console/latest/jupyter-notebook.pdf)]
-- [Documentation for Project Jupyter](http://jupyter.readthedocs.org/en/latest/index.html) [[PDF](https://media.readthedocs.org/pdf/jupyter/latest/jupyter.pdf)]
+- [Documentation for Jupyter Console](https://jupyter-console.readthedocs.io/en/latest/) [[PDF](https://media.readthedocs.io/pdf/jupyter-console/latest/jupyter-notebook.pdf)]
+- [Documentation for Project Jupyter](https://jupyter.readthedocs.io/en/latest/index.html) [[PDF](https://media.readthedocs.io/pdf/jupyter/latest/jupyter.pdf)]
```

Comment:
Again, these `media.` links do not work with the `.io` domain. The top one was actually already broken, though...